message MessageServantUpdate
{
    required int32 status = 1;
}
